### PR TITLE
Added files for building on kathleen

### PR DESCRIPTION
**Pull request by tkoskela**

### CODE REVIEW

File: modules/modules.kathleen.intel [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+module load mpi/intel/2018/update3
+module load python2/recommended
+module load cmake/3.13.3
+module load libtool/2.4.6
+module load autoconf/2.69
+module load automake/1.16.1
+module load hdf/5-1.10.2-impi/intel-2018
+module load metis/5.1.0/intel-2015-update2
+module load fftw/2.1.5/intel-2015-update2
+#module load fftw/3.3.8-impi/intel-2018
```
Comment (giordano) [inline modules/modules.kathleen.intel:13]: Why is this commented?

File: modules/modules.kathleen.intel [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+module load gsl/2.4/intel-2017
```
Comment (giordano) [inline modules/modules.kathleen.intel:14]: Missing the newline at the end of the file :upside_down_face:

File: modules/modules.kathleen.intel (hunk 1 above)
Comment (tkoskela) [reply modules/modules.kathleen.intel:13]: Becuase the previous line loads ffwt/2 and I was not sure if we explicitly need it for some legacy reason.

File: modules/modules.kathleen.intel (hunk 2 above)
Comment (tkoskela) [reply modules/modules.kathleen.intel:14]: 🙄

**Comment by giordano** [comment]
Should we create a branch out of `master`, make it the default and use that as target for this PR?

**Comment by tkoskela** [comment]
Why not merge to master?

**Comment by giordano** [comment]
In case we want to merge these changes back to upstream.  No idea if we want to

**Comment by tkoskela** [comment]
Fair enough. What should we call it? `ucl-kathleen`?

**Pull request merged**